app/frontend/src/features/messages/groupchats/MembersDialog.tsx

```diff
@@ -87,29 +85,31 @@ function MemberListItem({
   const handleRemoveAdmin = () => removeAdmin.mutate();
 
   return (
-    <ListItem dense>
-      <ListItemAvatar className={classes.avatarContainer}>
-        <Avatar user={member} className={classes.avatar} />
-      </ListItemAvatar>
-      <ListItemText
-        disableTypography
-        className={classes.memberListItemTextContainer}
-      >
-        <TextBody>
-          {member.name}
-          {memberIsAdmin && " (admin)"}
-        </TextBody>
-        {userIsAdmin &&
-          (memberIsAdmin ? (
-            <Button loading={removeAdmin.isLoading} onClick={handleRemoveAdmin}>
-              Remove as admin
-            </Button>
-          ) : (
-            <Button loading={makeAdmin.isLoading} onClick={handleMakeAdmin}>
-              Make admin
-            </Button>
-          ))}
-      </ListItemText>
+    <ListItem dense className={classes.memberListItemContainer}>
+      {userIsAdmin &&
+        //TODO: Colours
+        (memberIsAdmin ? (
+          <IconButton
```

Comment:
```suggestion
          <IconButton
            aria-label="Remove as admin"
```
We should add the text we removed in favour of icon buttons as `aria-label` for these instead so they're still accessible to non-visual users.